src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -363,7 +566,7 @@ public Reconciler(string modelFile)
                 Contracts.Assert(toOutput.Length == 1);
 
                 var outCol = (OutColumn)toOutput[0];
```

Comment:
>var outCol = (OutColumn)toOutput[0]; [](start = 16, length = 36)

Should we extend this also? It's only using 1 output.

Reply:
TF seems to have the same  reconciler logic. It works fine as-is, although we should double check it.

[https://github.com/dotnet/machinelearning/blob/cb9effcd091c60fa291aad96cc18c14ddf841b6f/src/Microsoft.ML.TensorFlow/TensorflowTransform.cs#L1237](https://github.com/dotnet/machinelearning/blob/cb9effcd091c60fa291aad96cc18c14ddf841b6f/src/Microsoft.ML.TensorFlow/TensorflowTransform.cs#L1237)

---
In reply to: [232840804](https://github.com/dotnet/machinelearning/pull/1586#discussion_r232840804) [](ancestors = 232840804)